kv/kv.go

```diff
@@ -119,26 +119,51 @@ const (
 	PresumeKeyNotExists
 )
 
-// Transaction defines the interface for operations inside a Transaction.
-// This is not thread safe.
-type Transaction interface {
-	// Get gets the value for key k from KV store.
+// MemStore is an abstract KV storage.
+type MemStore interface {
```

Comment:
Why not name it KVStore?
It is an interface, implementation can be pesistent store.